CONTRIBUTOR-LADDER.md

```diff
@@ -0,0 +1,129 @@
+    * Report and sometimes resolve issues
+    * Occasionally submit PRs
+    * Writing a CFP
+    * Contribute to the documentation
+    * Show up at community meetings and/or leads and/or takes notes
+    * Answer questions from other community members
+    * Submit feedback and comments on issues and PRs
+    * Test releases and patches and submit reviews
+    * Promote the project in public
+
+Non-code contributions can be documented by making a PR to add yourself to CONTRIBUTORS.md. 
+
+* Privileges:
+    * Ability to be assigned to Issues and ask for Reviews
+ 
+Becoming a Community Contributor is the first step towards becoming an Organization Member
+
+### Organization Member
+Description: An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges in project repositories and as such are expected to act in the interests of the whole project. Organization Members can take on additional roles within the project and can work more independently, for example triggering CI. 
+
+An Organization Member has all the rights and responsibilities of a Community Contributor, plus:
+
+* Responsibilities:
+    * Continues to contribute regularly
+    * Work to accomplish the tasks they volunteer to do within the project
+
+* Qualifications:
+    * Must have successful contributions to the project, including at least one of the following:
+        * Written and/or reviewed PRs,
+        * Reviewed PRs,
+        * Made contributions that resolved Issues,
+        * Or some equivalent combination or contribution
+    * Must be actively contributing to at least one project area, where the privileges of Organization Member will be beneficial
+
+* Privileges:
+    * Can trigger CI
+    * Can leave reviews on PRs
+    * Can recommend other contributors to become Cilium Github Org Members
+    * Can nominate themselves to other roles within Cilium
+
+The process for a Contributor to become an Organization Member is as follows:
+
+1. Open a PR in the member management file
+2. Receive two "LGTM" comments from any Org Members, and approval by a committer who merges the PR
+
+### Reviewer
+Description: A Reviewer has responsibility for specific code, documentation, test, or other project areas. They are collectively responsible, with other Reviewers, for reviewing all changes to those areas and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.
+Reviewers are responsible for a "specific area." This can be a specific code directory, chapter of the docs, or other clearly-defined project component that is smaller than an entire repository or subproject. Most often it is one or a set of directories in one or more Git repositories. The "specific area" below refers to this area of responsibility.
+Reviewers have all the rights and responsibilities of an Organization Member, plus:
+* Responsibilities:
```

Comment:
```suggestion
Description: A Reviewer has responsibility for specific code, documentation, test, or other project areas. They are collectively responsible, with other Reviewers, for reviewing all changes to those areas and indicating whether those changes are ready to merge. They have a track record of contribution and review in the project.

Reviewers are responsible for a "specific area." This can be a specific code directory, chapter of the docs, or other clearly-defined project component that is smaller than an entire repository or subproject. Most often it is one or a set of directories in one or more Git repositories. The "specific area" below refers to this area of responsibility.
Reviewers have all the rights and responsibilities of an Organization Member, plus:

* Responsibilities:
```